src/cstubs_applicative.opam

```diff
@@ -0,0 +1,22 @@
+opam-version: "2"
```

Comment:
Our other .opam files should be this nice.

Reply:
Thanks. I plan to use this in the `zexe` submodule, so it needed to be `opam pin`-able for standalone use there, and it only seemed right to do it properly!